### PR TITLE
Move features training into index file

### PR DESCRIPTION
**Pull request by esthermmoturi**
This pull request is to move the [training page ](https://docs.communityhealthtoolkit.org/building/features/training/)from the 'Features' section and make it the index page under '[Training cards section](https://docs.communityhealthtoolkit.org/building/training/training-cards/)'. This is an effort to compile related content within the docsite. It also includes aliases added, fixing of links and movement of images. [This](https://docs.communityhealthtoolkit.org/building/training/training-cards/) page was also renamed to Training Cards Configuration.

### CODE REVIEW

File: content/en/building/training/training-cards-resources/_index.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 title: "Training Cards Resources"
 linkTitle: "Training Cards Resources"
-weight: 15
+weight: 3
 description: >
   Training cards for CHT
 keyword: training
 relatedContent: >
-  building/features/training
+  building/training/training-cards
```
Comment (andrablaj) [inline content/en/building/training/training-cards-resources/_index.md:9]: This line is duplicated. I believe it should be removed?

File: content/en/building/training/training-cards/_index.md [hunk 2]
```diff
@@ -1,141 +1,45 @@
...
+  building/training/training-cards-resources
   building/examples/training
   building/examples/learning-care/
 aliases:
-   - /building/guides/training/training-cards/
-   - /apps/guides/training/training-cards/
+   - /building/features/training/
+   - /apps/features/training/
 ---
-_Introduced in 4.2.0_
```
Comment (andrablaj) [inline content/en/building/training/training-cards/_index.md:17]: I would add this line on the `building/training/training-cards/` page, as that is the first link the readers might bump into.

File: content/en/building/training/training-cards/training-cards-configuration.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+description: >
+  Deploy in-app training cards for remote training.
+keyword: training
+relatedContent: >
+  building/training/training-cards
+  building/training/training-cards-resources/
+  building/examples/training
+  building/examples/learning-care/
+aliases:
+   - /building/training/training-cards/
```
Comment (andrablaj) [inline content/en/building/training/training-cards/training-cards-configuration.md:14]: I don't think the first `alias` is necessary, as this is the page still exists, but the content was replaced the `Training` content, right? So the readers who have that link should land on the correct page, which is the one you replaced.

**Review by andrablaj — COMMENTED** [review]
Just some small questions, mostly related to `aliases`.

File: content/en/building/training/training-cards-resources/_index.md (hunk 1 above)
Comment (esthermmoturi) [reply content/en/building/training/training-cards-resources/_index.md:9]: Done!

File: content/en/building/training/training-cards/_index.md (hunk 2 above)
Comment (esthermmoturi) [reply content/en/building/training/training-cards/_index.md:17]: Done!

File: content/en/building/training/training-cards/training-cards-configuration.md (hunk 3 above)
Comment (esthermmoturi) [reply content/en/building/training/training-cards/training-cards-configuration.md:14]: Removed!

**Pull request merged**